Plugins/Public/playercntl_plugin/PurchaseRestrictions.cpp

```diff
@@ -236,6 +260,28 @@ namespace PurchaseRestrictions
 			}
 		}
 
+		// Restrict things via player tag
+		if (set_bCheckTagRestrictions)
+		{
+			if (mapTagList.find(gbi.iGoodID) != mapTagList.end())
+			{
+				list<string> tags = mapTagList[gbi.iGoodID];
+				string scCharname = wstos((const wchar_t*)Players.GetActiveCharacterName(iClientID));
+				bool validTag = find(tags.begin(), tags.end(), scCharname) != tags.end();
```

Comment:
This appears to make it such that the check is not for tags, the actual whole character name must be in the config?